package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "github-action-for-generator",
+  "version": "0.0.2",
```

Comment:
Should it be `0.1.0` or even `0.2.0`?

Reply:
you are right, 0.1.0, I think I crafted it in decent way that even 1.0.0. would be fine, but generator is not 1.0 and 0.1 feels more appropriate